<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pardus Testing Framework</string>
  </property>
  <widget class="QGroupBox" name="group_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>20</red>
         <green>19</green>
         <blue>18</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>20</red>
         <green>19</green>
         <blue>18</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QTextEdit" name="text_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="package_label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>26</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="text_observation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>531</width>
     <height>70</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Finish</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>420</y>
     <width>555</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Did the above test work as expected?</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QRadioButton" name="yes_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Yes</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QRadioButton" name="unable_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Unable to peform this test</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_observation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter your observation: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8">
     <widget class="QRadioButton" name="no_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;No</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>yes_button</tabstop>
  <tabstop>unable_button</tabstop>
  <tabstop>text_observation</tabstop>
  <tabstop>next_button</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>quit_button</tabstop>
  <tabstop>text_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>yes_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_observation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>text_observation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yes_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_observation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_observation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_observation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unable_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_observation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unable_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_observation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yes_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>clear_button</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>614</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unable_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>clear_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>clear_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unable_button</sender>
   <signal>clicked()</signal>
   <receiver>text_observation</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked()</signal>
   <receiver>text_observation</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>text_edit</sender>
   <signal>textChanged()</signal>
   <receiver>yes_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
